### PR TITLE
Iranian-Turanian colonial cultures

### PR DESCRIPTION
**Pull request by IohannesIohannium**
As in #307

### CODE REVIEW

File: EU4toV2/Data_Files/blankMod/output/localisation/0_Cultures.csv [hunk 1]
```diff
@@ -133,4 +133,7 @@ afro_carpathian;Afro-Carpathian;Afro-carpate;Afro-Karpaten;Afro-karpacki;Afro-c
 afro_italian;Afro-Italian;Afro-italien;Afro-Italienisch;Afro-w�oski;Afro-italiano;Afro-italiano;Afro-olasz;Afro-ital�tina;Afro-olasz;Afro-Italiaanse;Afro-italiano;����-�����������;Afro-italialainen;x
 afro_greek;Afro-Byzantine;Afro-byzantine;Afro-Byzantinisch;Afro-bizantyjski;Afro-bizantino;Afro-bizantino;Afro-biz�nci;Afro-byzantsk�;Afro-biz�nci;Afro-Byzantijnse;Afro-bizantino;����-������������;Afro-Bysantin;x
 afro_slavic;Afro-Slavic;Afro-slave;Afro-Slawisch;Afro-s�owia�ski;Afro-eslavo;Afro-slavo;Afro-szl�v;Afro-slovansk�;Afro-szl�v;Afro-Slavische;Afro-eslavo;����-����������;Afro-slaavilainen;x
-aromanian;Aromanian;Aroumain;Aromunisch;Arumu�ski;Arrumano;Arumeno;Arom�n;Arumun�tina;Arom�n;Aroemeens;Arromeno;����������;Aromania;x
\ No newline at end of file
+aromanian;Aromanian;Aroumain;Aromunisch;Arumu�ski;Arrumano;Arumeno;Arom�n;Arumun�tina;Arom�n;Aroemeens;Arromeno;����������;Aromania;x
+yangichi;Yangichi;Yangichi;Yangichi;Yangichi;Yangichi;Yangichi;Yangichi;Yangichi;Yangichi;Yangichi;Yangichi;�������;Yangichi;x
+saahi;Janubi;Janubi;Janubi;Janubi;Janubi;Janubi;Janubi;Janubi;Janubi;Janubi;Janubi;��������;Janubi;x
```
Comment (Idhrendur) [inline EU4toV2/Data_Files/blankMod/output/localisation/0_Cultures.csv]: janubi;...?

File: EU4toV2/Data_Files/blankMod/output/localisation/0_Cultures.csv [hunk 2]
```diff
@@ -133,4 +133,7 @@ afro_carpathian;Afro-Carpathian;Afro-carpate;Afro-Karpaten;Afro-karpacki;Afro-c
...
 afro_greek;Afro-Byzantine;Afro-byzantine;Afro-Byzantinisch;Afro-bizantyjski;Afro-bizantino;Afro-bizantino;Afro-biz�nci;Afro-byzantsk�;Afro-biz�nci;Afro-Byzantijnse;Afro-bizantino;����-������������;Afro-Bysantin;x
 afro_slavic;Afro-Slavic;Afro-slave;Afro-Slawisch;Afro-s�owia�ski;Afro-eslavo;Afro-slavo;Afro-szl�v;Afro-slovansk�;Afro-szl�v;Afro-Slavische;Afro-eslavo;����-����������;Afro-slaavilainen;x
-aromanian;Aromanian;Aroumain;Aromunisch;Arumu�ski;Arrumano;Arumeno;Arom�n;Arumun�tina;Arom�n;Aroemeens;Arromeno;����������;Aromania;x
\ No newline at end of file
+aromanian;Aromanian;Aroumain;Aromunisch;Arumu�ski;Arrumano;Arumeno;Arom�n;Arumun�tina;Arom�n;Aroemeens;Arromeno;����������;Aromania;x
+yangichi;Yangichi;Yangichi;Yangichi;Yangichi;Yangichi;Yangichi;Yangichi;Yangichi;Yangichi;Yangichi;Yangichi;�������;Yangichi;x
+saahi;Janubi;Janubi;Janubi;Janubi;Janubi;Janubi;Janubi;Janubi;Janubi;Janubi;Janubi;��������;Janubi;x
+saahi;Afrigi;Afrigi;Afrigi;Afrigi;Afrigi;Afrigi;Afrigi;Afrigi;Afrigi;Afrigi;Afrigi;�������;Afrigi;x
```
Comment (Idhrendur) [inline EU4toV2/Data_Files/blankMod/output/localisation/0_Cultures.csv]: afrigi;...?

File: EU4toV2/Data_Files/blankMod/output/events/postColonial.txt [hunk 3]
```diff
@@ -1,296 +1,293 @@
...
-country_event = {  id = 251053  title = "EVTNAME251053"  desc = "EVTDESC251001"  major = yes  picture = "Riot"    trigger = {   is_independant = yes   capital_scope = {    OR = {     continent = europe     continent = asia    }   }   NOT = { has_global_flag = SON_independence }   has_global_flag = der_postcolonialism   is_possible_vassal = SON   NOT = { exists = SON }  }    mean_time_to_happen = {   months = 48   modifier = {    factor = 0.75    year = 1845   }   modifier = {    factor = 0.75    year = 1855   }   modifier = {    factor = 0.75    year = 1860   }   modifier = {    factor = 0.5    year = 1865   }  }    immediate = {   set_global_flag = SON_independence   any_owned = { limit = { is_core = SON } secede_province = SON remove_core = THIS }   SON = {    leadership = 80    years_of_research = 5    set_country_flag = buro_independence    election = yes   }  }   option = {   name = "EVTOPTA251001"   create_vassal = SON   ai_chance = { factor = 25 }  }    option = {   name = "EVTOPTB251001"   war = {    target = SON    attacker_goal = {     casus_belli = conquest    }       defender_goal = {     casus_belli = become_independent    }
```
Comment (Idhrendur) [inline EU4toV2/Data_Files/blankMod/output/events/postColonial.txt]: It looks like this either incorporates multiple changes, or somehow dropped some other ones (Oranje, Transvaal, and Natalia are missing here, plus many other changes later). Can you double-check this file? If those changes are intended, can you put them in a separate PR for the sake of tracking?

File: EU4toV2/Data_Files/blankMod/output/common/cultures.txt [hunk 4]
```diff
@@ -1,4716 +1,4758 @@
...
-		Cosmin Daniel Dumitru Emanuel Eremia Eugen Ferdinand Flaviu Florea Gheorghe
-		Grigore Henric Horatiu Iacob Ilie Ioan Ion "Ion Emanuel" Iosif Iulian
-		Iuliu Laurentiu Leontin Luca Lucian Mihai Mircea Nicolae Ovidiu Petru
-		Radu Sergiu Silviu Simion Stefan Valeriu Vasile Victor Vintila Vlad
-		}
-
-		last_names = { Angelescu Antonescu Argetoianu Averescu Baldescu Barozzi Berindei Cernat Christecu Cihoski
-		Ciuperca Coanda Crainiceanu Critescu Culcer Cuza "de Hohenzollern" Dona Dragalina Dumitrescu
-		Falcoianu Florescu Gardescu Grigorescu Froza Harjeu Holban Iancovescu Iliasevici Iliescu
-		Ionascu Ionescu Ionitiu Koslinski Lahovary
```
[the remark this answers is not in the document]
Comment (Idhrendur) [reply EU4toV2/Data_Files/blankMod/output/common/cultures.txt]: Should have been on

    dinajidad = { #Pishgam

[the remark this answers is not in the document]
Comment (Idhrendur) [reply EU4toV2/Data_Files/blankMod/output/common/cultures.txt]: Should have been on

    janub = { #Nimruzi

[the remark this answers is not in the document]
Comment (Idhrendur) [reply EU4toV2/Data_Files/blankMod/output/common/cultures.txt]: Should have been on 

    saahi = { #Afrighayi

File: EU4toV2/Data_Files/cultureMap.txt [hunk 5]
```diff
@@ -1,1130 +1,1136 @@
...
-link = { vic2 = qalindigi eu4 = chorasmian eu4 = sabir eu4 = hunnic eu4 = mongol eu4 = chahar eu4 = khalkha eu4 = oirats eu4 = khitan eu4 = altaic_colonial eu4 = khalkas eu4 = khorchin eu4 = tumed eu4 = uriankhai eu4 = siberian eu4 = khanty eu4 = chukchi eu4 = mansi eu4 = mari eu4 = nenet eu4 = nivkh eu4 = selkup eu4 = yakut eu4 = yukagyr eu4 = buryat eu4 = tungus eu4 = daur eu4 = eveni eu4 = evenki eu4 = jurchen eu4 = oroqen eu4 = yukaghir eu4 = tibetan eu4 = balti eu4 = bhutanese eu4 = ladakhi eu4 = tibetic_colonial region = niger_region region = guinea_region region = central_africa_region region = sahel_region region = horn_of_africa_region region = east_africa_region region = kongo_region region = south_africa_region }
-link = { vic2 = azerbaijani eu4 = azerbadjani eu4 = azerbaijani }
-link = { vic2 = hazara eu4 = mongol religion = shiite }
-link = { vic2 = turkmen eu4 = turkmeni }
-link = { vic2 = mongol eu4 = mongol eu4 = chahar eu4 = khalkha eu4 = oirats eu4 = khitan }
-link = {
```
[the remark this answers is not in the document]
Comment (Idhrendur) [reply EU4toV2/Data_Files/cultureMap.txt]: Should have been on

    # Iranian - persian, east_persian, baluchi
    # and colonial Iranian-Turanian

File: EU4toV2/Data_Files/blankMod/output/localisation/0_Cultures.csv (hunk 1 above)
Comment (Idhrendur) [reply EU4toV2/Data_Files/blankMod/output/localisation/0_Cultures.csv]: The last line there

File: EU4toV2/Data_Files/blankMod/output/localisation/0_Cultures.csv (hunk 2 above)
Comment (Idhrendur) [reply EU4toV2/Data_Files/blankMod/output/localisation/0_Cultures.csv]: The last line showed there.

Comment (IohannesIohannium) [reply EU4toV2/Data_Files/blankMod/output/localisation/0_Cultures.csv]: Fixed

File: EU4toV2/Data_Files/blankMod/output/common/cultures.txt (hunk 4 above)
[the remark this answers is not in the document]
Comment (IohannesIohannium) [reply EU4toV2/Data_Files/blankMod/output/common/cultures.txt]: Together with `janub` and `saahi`, as stated on Github, changing the string would make it more difficult to keep compatibility with previous versions of the mod. If such thing is irrelevant, I'll take care of such on another pull.

File: EU4toV2/Data_Files/blankMod/output/events/postColonial.txt (hunk 3 above)
Comment (IohannesIohannium) [reply EU4toV2/Data_Files/blankMod/output/events/postColonial.txt]: Fixed, I had used an outdated base

File: EU4toV2/Data_Files/blankMod/output/common/cultures.txt (hunk 4 above)
[the remark this answers is not in the document]
Comment (Idhrendur) [reply EU4toV2/Data_Files/blankMod/output/common/cultures.txt]: That kind of compatibility isn't really relevant to EU4 to Vic2.
However, it means Vic2 to HoI4 should map both the old and the new culture to graphics. But it seems more maintainable/understandable to do it that way, I think.

And I'd kind of rather have it in this PR, as that keeps all the changes together. I know that makes a bit more work for you, sorry!

[the remark this answers is not in the document]
Comment (IohannesIohannium) [reply EU4toV2/Data_Files/blankMod/output/common/cultures.txt]: Okay, fixed it

File: EU4toV2/Data_Files/blankMod/output/common/cultures.txt [hunk 6]
```diff
@@ -615,27 +615,27 @@ italian = {
...
-			Andriai A�echui A�erestui Anto�inui Anto�ui A�ui Aporo�ui Arcadzui Archemnui Artapsiai Auetui Augstinui
-			Augstui Aurelleanui Aurellui Bartoromeui Ba�ellui Bechellui Bellesariui Be�acui Be�edeptui Betumnatui
-			Bra�od�enui Bre�ui Breta�ecui Brezui Buld�ui Camurod�enui Carinui Carui Ca�ui Centui Ch�chellui Ch�llestinui
-			Cheprianui Cher�i Ch�sa Chetsiui Chetui Chodo�ui Chuspai Ciaudzui Ciementsui Cometai Consta�i Constantinui
-			Constantsui Cor�ellui Crasui Dadacui Dariui Deca�ui Dechemui Dechui Dentupei Desachentui Domechanui Dome�ecui
-			Dometsui Drusui D�era�ui D�ermanui D�esatorecui D��ui �d�edzui Ellanui Elpedzui �melleanui �mellui Emnatsui
-			E�odzui Eposomnatui Eptapori Ercullui Esarnui �tsui Eubarnacui Fabeanui Fabiui Fabrichui Faran�i Faustui
-			Fellepui Felli Flaueanui Floreanui Gallenui Galleriui Garui Gordeanui Grachanui Gracui Ioa�i Ioueanui
-			Iulleanui Iullui Iu�ui Iusche�eanui Iustinui Laurentsui Lleberiui Lleche�ui Llo Llontsui Luchui Madacui
+		first_names = { Abucatui Achemnatui Acheporecui Adrian Albinui Albui Allepsandri Ambarui Ambroui Anaronacui Anastaui
```
Comment (Idhrendur) [inline EU4toV2/Data_Files/blankMod/output/common/cultures.txt]: Starting about here, a lot of characters start changing in the names. Was that intentional?

Comment (IohannesIohannium) [reply EU4toV2/Data_Files/blankMod/output/common/cultures.txt]: Absolutely not, but Github doesn't seem to like anything outside Basic Latin

Comment (Idhrendur) [reply EU4toV2/Data_Files/blankMod/output/common/cultures.txt]: Got it. I think I can put together a fix, but it's late so it'll have to be tomorrow or Saturday.

**Comment by IohannesIohannium** [comment]
By error, I added another addition to this pull request instead of making it separate. This contains now also the new Roman hybrid cultures and a fix to Copts

**Pull request merged**